ocs_ci/ocs/platform_nodes.py

```diff
@@ -812,7 +812,7 @@ def get_existing_indexes(self, index_list):
         """
         temp = []
         for index in index_list:
-            temp.append(int(index.split('-')[1][2:]))
+            temp.append(int(index.split('-')[-1][2:]))
```

Comment:
this could work just fine if there is no chance that we will have double digit number here.
Can we have double digit numbe like num12 at some point?

Reply:
@clacroix12 , what if in scale testing we cross double digits ?
```
mystr = "jnk-au3c33-t1-num22"
```
then we will get only 2.

Another approach I can think of is 
```python
In [111]: mystr1                                                                                                                        
Out[111]: 'jnk-au3c33-t1-nu12m23'

In [112]: int(re.findall(r'\d+', mystr1.split('-')[-1])[-1])                                                                            
Out[112]: 23

```

Reply:
Double digit index is a really good point, I think the use of regex that you have @shylesh is probably the best solution.

Reply:
@clacroix12 @shylesh @petr-balogh since stack name has always suffix with "no{index}"  https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/aws.py#L943

I feel its safe to use ``` int(index.split('-')[-1][2:])``` , it handles double digit also and I am also ok using Shylesh approach as well.

Reply:
I like more the reg exp approach here.

Reply:
Done